src/common/models/data-cube/data-cube.ts

```diff
@@ -339,7 +343,9 @@ export class DataCube implements Instance<DataCubeValue, DataCubeJS> {
       defaultPinnedDimensions: parameters.defaultPinnedDimensions ? OrderedSet(parameters.defaultPinnedDimensions) : null,
       maxSplits: parameters.maxSplits,
       maxQueries: parameters.maxQueries,
-      refreshRule
+      refreshRule,
+      warningTitle: parameters.warningTitle,
+      warningDescription: parameters.warningDescription,
```

Comment:
Fields warningTitle and warningDescription are duplicates of title and description. If Turnilo administrator has to add some additional information to data cube he/she should edit existing title/description.

If additional restrictions are really needed we should consider boolean flags like: enabled: [true|false] (true by default to be backward compatible).

With the proposed design we end up with: warningTitle, errorTitle, holidayTitle, etc ;)